install_golangci-lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0
```

Comment:
What about making this a target in the `Makefile` instead of a script? Should be quite easy to have as a make target right?